test/test_browser.py

```diff
@@ -2678,6 +2723,11 @@ def test_doublestart_bug(self):
   })
   @requires_threads
   def test_html5_core(self, opts):
+    if self.is_wasm64():
+      if '-sMIN_CHROME_VERSION=0' in opts:
+        self.skipTest('wasm64 doesnt support older browsers')
+      if '-sPROXY_TO_PTHREAD' in opts:
+        self.skipTest('_emscripten_set_keypress_callback_on_thread broken')
```

Comment:
Is this related to wasm64?

Reply:
Yes _emscripten_set_keypress_callback_on_thread doesn't work on wasm64.. I'll update the comment